### PR TITLE
Implementation for argmin in jax frontend

### LINKED ISSUE #13887: argmin

### PR DESCRIPTION
**Pull request by felixreynoso**
To close #13887

### CODE REVIEW

File: ivy/functional/frontends/jax/numpy/searching_sorting.py [hunk 1]
```diff
@@ -32,6 +32,10 @@ def argmax(a, axis=None, out=None, keepdims=False):
     return ivy.argmax(a, axis=axis, keepdims=keepdims, out=out, dtype=ivy.int64)
 
 
+def argmin(a, axis=None, out=None, keepdims=False):
```
Comment (valeriocost) [inline ivy/functional/frontends/jax/numpy/searching_sorting.py:35]: I think you misspelled here 😅 argmin with argmax

Comment (felixreynoso) [reply ivy/functional/frontends/jax/numpy/searching_sorting.py:36]: Hello, just fixed it. Thank you for the observation.

**Comment by valeriocost** [comment]
The tests appear to be failing on the CI actions, are they passing locally on your workspace? also, make sure to not include in your commit any file that is not related to the task you are working on (in this case the .idea folder), thanks! 😄

**Comment by felixreynoso** [comment]
Do you mean these tests? 

Screen Shot 2023-04-12 at 7 21 54 AM

This is my first time contributing to a large project like this, so any guidance is greatly appreciated.

**Comment by valeriocost** [comment]
Yes, if you go into the different "intelligent-test" and then in the "Determine and Run test section" you can see all the functions that are failing for each test. There is also the function you are working on so I was asking if you ran the test on your machine to see if it was working there. Let me know if you have any questions 😄

**Comment by felixreynoso** [comment]
got it, I will check it out and recommit when the tests pass. 

Thanks for the observation.

**Comment by valeriocost** [comment]
No problem! 😄

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Pull request closed without merging**